app/build.gradle.kts

```diff
@@ -61,6 +61,25 @@ android {
         }
     }
 
+    flavorDimensions += "variant"
+    productFlavors {
+        create("dark") {
+            dimension = "variant"
+            resValue("string", "apps_name", "Lawnicons")
+        }
+        create("light") {
+            dimension = "variant"
+            applicationIdSuffix = ".light"
+            versionNameSuffix = "-light"
+            resValue("string", "apps_name", "Lawnicons (light)")
+        }
+    }
+    sourceSets.named("dark") {
+        res.srcDirs("src/dark/res/")
+    }
+    sourceSets.named("light") {
+        res.srcDirs("src/light/res/")
+    }
```

Comment:
```suggestion
```
Unnecessary, they are the default paths.

Reply:
Is it so? I was not aware of it. Thanks for suggestion